modules/rubiconBidAdapter.js

```diff
@@ -368,26 +282,84 @@ function RubiconAdapter() {
           return memo;
         }, {'rpfl_elemid': bidRequest.placementCode});
 
-      try {
-        bidmanager.addBidResponse(bidRequest.placementCode, bid);
-      } catch (err) {
-        utils.logError('Error from addBidResponse', null, err);
-      }
-    });
-    // Run the Emily user sync
-    hasUserSyncFired = syncEmily(hasUserSyncFired);
+      bids.push(bid);
+
+      return bids;
+    }, []);
+
+    return bids;
+  },
+  getUserSyncs: function() {
+    if (!hasSynced) {
```

Comment:
Not sure if I understand the business use-cases behind this, but... is there any reason why someone would want to sync pixels more than once?

If not, could it be pulled down into the user sync module? Trying to reduce the number of things that adapters need to worry about.

Reply:
I think there's some use for that; although it'd have to be a configuration option I think in the case that a bidder wanted to fire multiple syncs to the same endpoint for whatever reason.

It's a good suggestion, maybe we should open that as a separate issue?  We are hoping to get these changes in the next release.

Reply:
yeah, that's fine for now.

Be careful leaving it for too long, though. Adapters will look towards these first few files as an example for how to build their own... so any tech debt will accrue interest very quickly over the next few months.